charts/kyuubi/values.yaml

```diff
@@ -116,6 +116,16 @@ server:
       nodePort: ~
       annotations: {}
 
+  # Exposes metrics in Prometheus format
+  prometheus:
+    enabled: true
+    port: 10019
+    service:
+      type: ClusterIP
+      port: "{{ .Values.server.prometheus.port }}"
+      nodePort: ~
+      annotations: {}
```

Comment:
I cannot find how these values are used. Did you missed something in the commit?

Reply:
<img width="1170" alt="Screenshot 2023-08-07 at 10 45 50" src="https://github.com/apache/kyuubi/assets/43336508/f3e1d136-181d-4b19-ad30-746006eb8072">
it is wrapped in a loop

Reply:
Great! I missed that, sorry.